src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -148,14 +264,20 @@
          :body
          {:columns
           (let [column-data (fn [ws]
-                              (let [disabled? (= (:accessLevel ws) "NO ACCESS")]
+                              (let [no-access? (= (:accessLevel ws) "NO ACCESS")
+                                    disabled? (and no-access? (= (get-in ws [:workspace :authorizationDomain :membersGroupName])
+                                                                 (config/dbgap-authorization-domain)))]
                                 {:workspace-id (select-keys (:workspace ws) [:namespace :name])
                                  :href (let [x (:workspace ws)] (str (:namespace x) ":" (:name x)))
                                  :status (:status ws)
                                  :disabled? disabled?
-                                 :hover-text (when disabled? (if (= (get-in ws [:workspace :authorizationDomain :membersGroupName]) (config/dbgap-authorization-domain))
-                                                               dbGap-disabled-text
-                                                               non-dbGap-disabled-text))
+                                 ;; this will very soon return multiple auth domains, so im future-proofing it now
+                                 :auth-domains [(get-in ws [:workspace :authorizationDomain :membersGroupName])]
+                                 :no-access? no-access?
+                                 :hover-text (when no-access? (if (= (get-in ws [:workspace :authorizationDomain :membersGroupName])
+                                                                     (config/dbgap-authorization-domain))
```

Comment:
Just checking: Does this value meet our criteria for "belongs in config?" If not, you could just `def` it in the same file so it is available throughout the project.